sdk/src/Services/S3/Custom/Util/_bcl/S3PostUploadRequest.cs

```diff
@@ -40,7 +40,8 @@ namespace Amazon.S3.Util
     /// This includes metadata. If metadata is specified in the policy, then it must be included in the request. Adding metadata not in the
     /// policy will cause the POST to fail.
     /// </para><para>
-    /// For more information, <see href="http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingHTTPPOST.html"/>
+    /// For more information, <see href="http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingHTTPPOST.html"/> or the API documentation at
```

Comment:
Can you update the summary for this class (on line 35) to say `Parameters for uploading to Amazon S3 a file using HTTP POST`? Instead of what we have right now (`Parameters for uploading to Amazon S3 a file using HTTP POS`...)

Reply:
Made the change.